platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -205,7 +206,7 @@ private boolean deliverMessages() {
 
   /**
    * Constructor
-   *
+   * <p>
```

Comment:
this should be removed, was added by Android Studio but will break our javadoc generation.